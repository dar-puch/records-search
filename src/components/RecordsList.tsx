import React from "react";
import { Release } from "../utils/api";
import {
  Box,
  GridList,
  GridListTile,
  Card,
  CardActionArea,
  CardMedia,
  CardContent,
  Typography,
} from "@material-ui/core";

interface IRecordsList {
  list: Release[];
}

export const RecordsList: React.FC<IRecordsList> = ({ list }) => (
  <GridList cellHeight={350} cols={9} spacing={6}>
    {list.map((item, index) => (
      <GridListTile cols={2} key={index}>
        <Card>
          <CardActionArea>
            <CardMedia
              component="img"
              alt="Contemplative Reptile"
              height="200"
              width="200"
              image={item.cover_image}
              title={item.title}
            />
            <CardContent>
              <Typography gutterBottom={true} variant="body1">
                {item.title}
              </Typography>
              <Typography variant="body2" color="textSecondary">
                cat number: {item.catno} <br /> year: {item.year}
              </Typography>
            </CardContent>
          </CardActionArea>
        </Card>
      </GridListTile>
    ))}
  </GridList>
);
