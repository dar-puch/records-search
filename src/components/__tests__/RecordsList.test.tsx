import * as React from 'react';
import { shallow, mount } from 'enzyme';
import 'jest-styled-components';
import { RecordsList } from '../RecordsList';

describe('<RecordsList/>', () => {
  const list = [{ 'id': 1, "title": "Jason", "year": "1989", "catno": "ff45", "cover_image": "hhh" }, { 'id': 2, "title": "Olion", "year": "1955", "catno": "787r", "cover_image": "hhh" }];
  const wrapper = mount(<RecordsList list={list} />);
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('renders proper amount of list items', () => {
    expect(wrapper.find("li")).toHaveLength(2);
  });

});
