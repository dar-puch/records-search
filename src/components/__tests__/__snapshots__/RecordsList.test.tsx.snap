// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RecordsList/> renders correctly 1`] = `
<Component
  list={
    Array [
      Object {
        "catno": "ff45",
        "cover_image": "hhh",
        "id": 1,
        "title": "Jason",
        "year": "1989",
      },
      Object {
        "catno": "787r",
        "cover_image": "hhh",
        "id": 2,
        "title": "Olion",
        "year": "1955",
      },
    ]
  }
>
  <WithStyles(ForwardRef(GridList))
    cellHeight={320}
    cols={9}
    spacing={6}
  >
    <ForwardRef(GridList)
      cellHeight={320}
      classes={
        Object {
          "root": "MuiGridList-root",
        }
      }
      cols={9}
      spacing={6}
    >
      <ul
        className="MuiGridList-root"
        style={
          Object {
            "margin": -3,
          }
        }
      >
        <WithStyles(ForwardRef(GridListTile))
          cols={2}
          key=".$0"
          style={
            Object {
              "height": 326,
              "padding": 3,
              "width": "22.22222222222222%",
            }
          }
        >
          <ForwardRef(GridListTile)
            classes={
              Object {
                "imgFullHeight": "MuiGridListTile-imgFullHeight",
                "imgFullWidth": "MuiGridListTile-imgFullWidth",
                "root": "MuiGridListTile-root",
                "tile": "MuiGridListTile-tile",
              }
            }
            cols={2}
            style={
              Object {
                "height": 326,
                "padding": 3,
                "width": "22.22222222222222%",
              }
            }
          >
            <li
              className="MuiGridListTile-root"
              style={
                Object {
                  "height": 326,
                  "padding": 3,
                  "width": "22.22222222222222%",
                }
              }
            >
              <div
                className="MuiGridListTile-tile"
              >
                <WithStyles(ForwardRef(Card))
                  key=".0"
                >
                  <ForwardRef(Card)
                    classes={
                      Object {
                        "root": "MuiCard-root",
                      }
                    }
                  >
                    <WithStyles(ForwardRef(Paper))
                      className="MuiCard-root"
                      elevation={1}
                    >
                      <ForwardRef(Paper)
                        className="MuiCard-root"
                        classes={
                          Object {
                            "elevation0": "MuiPaper-elevation0",
                            "elevation1": "MuiPaper-elevation1",
                            "elevation10": "MuiPaper-elevation10",
                            "elevation11": "MuiPaper-elevation11",
                            "elevation12": "MuiPaper-elevation12",
                            "elevation13": "MuiPaper-elevation13",
                            "elevation14": "MuiPaper-elevation14",
                            "elevation15": "MuiPaper-elevation15",
                            "elevation16": "MuiPaper-elevation16",
                            "elevation17": "MuiPaper-elevation17",
                            "elevation18": "MuiPaper-elevation18",
                            "elevation19": "MuiPaper-elevation19",
                            "elevation2": "MuiPaper-elevation2",
                            "elevation20": "MuiPaper-elevation20",
                            "elevation21": "MuiPaper-elevation21",
                            "elevation22": "MuiPaper-elevation22",
                            "elevation23": "MuiPaper-elevation23",
                            "elevation24": "MuiPaper-elevation24",
                            "elevation3": "MuiPaper-elevation3",
                            "elevation4": "MuiPaper-elevation4",
                            "elevation5": "MuiPaper-elevation5",
                            "elevation6": "MuiPaper-elevation6",
                            "elevation7": "MuiPaper-elevation7",
                            "elevation8": "MuiPaper-elevation8",
                            "elevation9": "MuiPaper-elevation9",
                            "outlined": "MuiPaper-outlined",
                            "root": "MuiPaper-root",
                            "rounded": "MuiPaper-rounded",
                          }
                        }
                        elevation={1}
                      >
                        <div
                          className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
                        >
                          <WithStyles(ForwardRef(CardActionArea))>
                            <ForwardRef(CardActionArea)
                              classes={
                                Object {
                                  "focusHighlight": "MuiCardActionArea-focusHighlight",
                                  "focusVisible": "Mui-focusVisible",
                                  "root": "MuiCardActionArea-root",
                                }
                              }
                            >
                              <WithStyles(ForwardRef(ButtonBase))
                                className="MuiCardActionArea-root"
                                focusVisibleClassName="Mui-focusVisible"
                              >
                                <ForwardRef(ButtonBase)
                                  className="MuiCardActionArea-root"
                                  classes={
                                    Object {
                                      "disabled": "Mui-disabled",
                                      "focusVisible": "Mui-focusVisible",
                                      "root": "MuiButtonBase-root",
                                    }
                                  }
                                  focusVisibleClassName="Mui-focusVisible"
                                >
                                  <button
                                    className="MuiButtonBase-root MuiCardActionArea-root"
                                    disabled={false}
                                    onBlur={[Function]}
                                    onDragLeave={[Function]}
                                    onFocus={[Function]}
                                    onKeyDown={[Function]}
                                    onKeyUp={[Function]}
                                    onMouseDown={[Function]}
                                    onMouseLeave={[Function]}
                                    onMouseUp={[Function]}
                                    onTouchEnd={[Function]}
                                    onTouchMove={[Function]}
                                    onTouchStart={[Function]}
                                    tabIndex={0}
                                    type="button"
                                  >
                                    <WithStyles(ForwardRef(CardMedia))
                                      alt="Contemplative Reptile"
                                      component="img"
                                      height="150"
                                      image="hhh"
                                      title="Jason"
                                      width="150"
                                    >
                                      <ForwardRef(CardMedia)
                                        alt="Contemplative Reptile"
                                        classes={
                                          Object {
                                            "img": "MuiCardMedia-img",
                                            "media": "MuiCardMedia-media",
                                            "root": "MuiCardMedia-root",
                                          }
                                        }
                                        component="img"
                                        height="150"
                                        image="hhh"
                                        title="Jason"
                                        width="150"
                                      >
                                        <img
                                          alt="Contemplative Reptile"
                                          className="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img"
                                          height="150"
                                          src="hhh"
                                          title="Jason"
                                          width="150"
                                        />
                                      </ForwardRef(CardMedia)>
                                    </WithStyles(ForwardRef(CardMedia))>
                                    <WithStyles(ForwardRef(CardContent))>
                                      <ForwardRef(CardContent)
                                        classes={
                                          Object {
                                            "root": "MuiCardContent-root",
                                          }
                                        }
                                      >
                                        <div
                                          className="MuiCardContent-root"
                                        >
                                          <WithStyles(ForwardRef(Typography))
                                            gutterBottom={true}
                                            variant="body1"
                                          >
                                            <ForwardRef(Typography)
                                              classes={
                                                Object {
                                                  "alignCenter": "MuiTypography-alignCenter",
                                                  "alignJustify": "MuiTypography-alignJustify",
                                                  "alignLeft": "MuiTypography-alignLeft",
                                                  "alignRight": "MuiTypography-alignRight",
                                                  "body1": "MuiTypography-body1",
                                                  "body2": "MuiTypography-body2",
                                                  "button": "MuiTypography-button",
                                                  "caption": "MuiTypography-caption",
                                                  "colorError": "MuiTypography-colorError",
                                                  "colorInherit": "MuiTypography-colorInherit",
                                                  "colorPrimary": "MuiTypography-colorPrimary",
                                                  "colorSecondary": "MuiTypography-colorSecondary",
                                                  "colorTextPrimary": "MuiTypography-colorTextPrimary",
                                                  "colorTextSecondary": "MuiTypography-colorTextSecondary",
                                                  "displayBlock": "MuiTypography-displayBlock",
                                                  "displayInline": "MuiTypography-displayInline",
                                                  "gutterBottom": "MuiTypography-gutterBottom",
                                                  "h1": "MuiTypography-h1",
                                                  "h2": "MuiTypography-h2",
                                                  "h3": "MuiTypography-h3",
                                                  "h4": "MuiTypography-h4",
                                                  "h5": "MuiTypography-h5",
                                                  "h6": "MuiTypography-h6",
                                                  "noWrap": "MuiTypography-noWrap",
                                                  "overline": "MuiTypography-overline",
                                                  "paragraph": "MuiTypography-paragraph",
                                                  "root": "MuiTypography-root",
                                                  "srOnly": "MuiTypography-srOnly",
                                                  "subtitle1": "MuiTypography-subtitle1",
                                                  "subtitle2": "MuiTypography-subtitle2",
                                                }
                                              }
                                              gutterBottom={true}
                                              variant="body1"
                                            >
                                              <p
                                                className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
                                              >
                                                Jason
                                              </p>
                                            </ForwardRef(Typography)>
                                          </WithStyles(ForwardRef(Typography))>
                                          <WithStyles(ForwardRef(Typography))
                                            color="textSecondary"
                                            variant="body2"
                                          >
                                            <ForwardRef(Typography)
                                              classes={
                                                Object {
                                                  "alignCenter": "MuiTypography-alignCenter",
                                                  "alignJustify": "MuiTypography-alignJustify",
                                                  "alignLeft": "MuiTypography-alignLeft",
                                                  "alignRight": "MuiTypography-alignRight",
                                                  "body1": "MuiTypography-body1",
                                                  "body2": "MuiTypography-body2",
                                                  "button": "MuiTypography-button",
                                                  "caption": "MuiTypography-caption",
                                                  "colorError": "MuiTypography-colorError",
                                                  "colorInherit": "MuiTypography-colorInherit",
                                                  "colorPrimary": "MuiTypography-colorPrimary",
                                                  "colorSecondary": "MuiTypography-colorSecondary",
                                                  "colorTextPrimary": "MuiTypography-colorTextPrimary",
                                                  "colorTextSecondary": "MuiTypography-colorTextSecondary",
                                                  "displayBlock": "MuiTypography-displayBlock",
                                                  "displayInline": "MuiTypography-displayInline",
                                                  "gutterBottom": "MuiTypography-gutterBottom",
                                                  "h1": "MuiTypography-h1",
                                                  "h2": "MuiTypography-h2",
                                                  "h3": "MuiTypography-h3",
                                                  "h4": "MuiTypography-h4",
                                                  "h5": "MuiTypography-h5",
                                                  "h6": "MuiTypography-h6",
                                                  "noWrap": "MuiTypography-noWrap",
                                                  "overline": "MuiTypography-overline",
                                                  "paragraph": "MuiTypography-paragraph",
                                                  "root": "MuiTypography-root",
                                                  "srOnly": "MuiTypography-srOnly",
                                                  "subtitle1": "MuiTypography-subtitle1",
                                                  "subtitle2": "MuiTypography-subtitle2",
                                                }
                                              }
                                              color="textSecondary"
                                              variant="body2"
                                            >
                                              <p
                                                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                                              >
                                                cat number: 
                                                ff45
                                                 
                                                <br />
                                                 year: 
                                                1989
                                              </p>
                                            </ForwardRef(Typography)>
                                          </WithStyles(ForwardRef(Typography))>
                                        </div>
                                      </ForwardRef(CardContent)>
                                    </WithStyles(ForwardRef(CardContent))>
                                    <span
                                      className="MuiCardActionArea-focusHighlight"
                                    />
                                    <NoSsr>
                                      <WithStyles(undefined)
                                        center={false}
                                      >
                                        <ForwardRef(TouchRipple)
                                          center={false}
                                          classes={
                                            Object {
                                              "child": "MuiTouchRipple-child",
                                              "childLeaving": "MuiTouchRipple-childLeaving",
                                              "childPulsate": "MuiTouchRipple-childPulsate",
                                              "ripple": "MuiTouchRipple-ripple",
                                              "ripplePulsate": "MuiTouchRipple-ripplePulsate",
                                              "rippleVisible": "MuiTouchRipple-rippleVisible",
                                              "root": "MuiTouchRipple-root",
                                            }
                                          }
                                        >
                                          <span
                                            className="MuiTouchRipple-root"
                                          >
                                            <TransitionGroup
                                              childFactory={[Function]}
                                              component={null}
                                              exit={true}
                                            />
                                          </span>
                                        </ForwardRef(TouchRipple)>
                                      </WithStyles(undefined)>
                                    </NoSsr>
                                  </button>
                                </ForwardRef(ButtonBase)>
                              </WithStyles(ForwardRef(ButtonBase))>
                            </ForwardRef(CardActionArea)>
                          </WithStyles(ForwardRef(CardActionArea))>
                        </div>
                      </ForwardRef(Paper)>
                    </WithStyles(ForwardRef(Paper))>
                  </ForwardRef(Card)>
                </WithStyles(ForwardRef(Card))>
              </div>
            </li>
          </ForwardRef(GridListTile)>
        </WithStyles(ForwardRef(GridListTile))>
        <WithStyles(ForwardRef(GridListTile))
          cols={2}
          key=".$1"
          style={
            Object {
              "height": 326,
              "padding": 3,
              "width": "22.22222222222222%",
            }
          }
        >
          <ForwardRef(GridListTile)
            classes={
              Object {
                "imgFullHeight": "MuiGridListTile-imgFullHeight",
                "imgFullWidth": "MuiGridListTile-imgFullWidth",
                "root": "MuiGridListTile-root",
                "tile": "MuiGridListTile-tile",
              }
            }
            cols={2}
            style={
              Object {
                "height": 326,
                "padding": 3,
                "width": "22.22222222222222%",
              }
            }
          >
            <li
              className="MuiGridListTile-root"
              style={
                Object {
                  "height": 326,
                  "padding": 3,
                  "width": "22.22222222222222%",
                }
              }
            >
              <div
                className="MuiGridListTile-tile"
              >
                <WithStyles(ForwardRef(Card))
                  key=".0"
                >
                  <ForwardRef(Card)
                    classes={
                      Object {
                        "root": "MuiCard-root",
                      }
                    }
                  >
                    <WithStyles(ForwardRef(Paper))
                      className="MuiCard-root"
                      elevation={1}
                    >
                      <ForwardRef(Paper)
                        className="MuiCard-root"
                        classes={
                          Object {
                            "elevation0": "MuiPaper-elevation0",
                            "elevation1": "MuiPaper-elevation1",
                            "elevation10": "MuiPaper-elevation10",
                            "elevation11": "MuiPaper-elevation11",
                            "elevation12": "MuiPaper-elevation12",
                            "elevation13": "MuiPaper-elevation13",
                            "elevation14": "MuiPaper-elevation14",
                            "elevation15": "MuiPaper-elevation15",
                            "elevation16": "MuiPaper-elevation16",
                            "elevation17": "MuiPaper-elevation17",
                            "elevation18": "MuiPaper-elevation18",
                            "elevation19": "MuiPaper-elevation19",
                            "elevation2": "MuiPaper-elevation2",
                            "elevation20": "MuiPaper-elevation20",
                            "elevation21": "MuiPaper-elevation21",
                            "elevation22": "MuiPaper-elevation22",
                            "elevation23": "MuiPaper-elevation23",
                            "elevation24": "MuiPaper-elevation24",
                            "elevation3": "MuiPaper-elevation3",
                            "elevation4": "MuiPaper-elevation4",
                            "elevation5": "MuiPaper-elevation5",
                            "elevation6": "MuiPaper-elevation6",
                            "elevation7": "MuiPaper-elevation7",
                            "elevation8": "MuiPaper-elevation8",
                            "elevation9": "MuiPaper-elevation9",
                            "outlined": "MuiPaper-outlined",
                            "root": "MuiPaper-root",
                            "rounded": "MuiPaper-rounded",
                          }
                        }
                        elevation={1}
                      >
                        <div
                          className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
                        >
                          <WithStyles(ForwardRef(CardActionArea))>
                            <ForwardRef(CardActionArea)
                              classes={
                                Object {
                                  "focusHighlight": "MuiCardActionArea-focusHighlight",
                                  "focusVisible": "Mui-focusVisible",
                                  "root": "MuiCardActionArea-root",
                                }
                              }
                            >
                              <WithStyles(ForwardRef(ButtonBase))
                                className="MuiCardActionArea-root"
                                focusVisibleClassName="Mui-focusVisible"
                              >
                                <ForwardRef(ButtonBase)
                                  className="MuiCardActionArea-root"
                                  classes={
                                    Object {
                                      "disabled": "Mui-disabled",
                                      "focusVisible": "Mui-focusVisible",
                                      "root": "MuiButtonBase-root",
                                    }
                                  }
                                  focusVisibleClassName="Mui-focusVisible"
                                >
                                  <button
                                    className="MuiButtonBase-root MuiCardActionArea-root"
                                    disabled={false}
                                    onBlur={[Function]}
                                    onDragLeave={[Function]}
                                    onFocus={[Function]}
                                    onKeyDown={[Function]}
                                    onKeyUp={[Function]}
                                    onMouseDown={[Function]}
                                    onMouseLeave={[Function]}
                                    onMouseUp={[Function]}
                                    onTouchEnd={[Function]}
                                    onTouchMove={[Function]}
                                    onTouchStart={[Function]}
                                    tabIndex={0}
                                    type="button"
                                  >
                                    <WithStyles(ForwardRef(CardMedia))
                                      alt="Contemplative Reptile"
                                      component="img"
                                      height="150"
                                      image="hhh"
                                      title="Olion"
                                      width="150"
                                    >
                                      <ForwardRef(CardMedia)
                                        alt="Contemplative Reptile"
                                        classes={
                                          Object {
                                            "img": "MuiCardMedia-img",
                                            "media": "MuiCardMedia-media",
                                            "root": "MuiCardMedia-root",
                                          }
                                        }
                                        component="img"
                                        height="150"
                                        image="hhh"
                                        title="Olion"
                                        width="150"
                                      >
                                        <img
                                          alt="Contemplative Reptile"
                                          className="MuiCardMedia-root MuiCardMedia-media MuiCardMedia-img"
                                          height="150"
                                          src="hhh"
                                          title="Olion"
                                          width="150"
                                        />
                                      </ForwardRef(CardMedia)>
                                    </WithStyles(ForwardRef(CardMedia))>
                                    <WithStyles(ForwardRef(CardContent))>
                                      <ForwardRef(CardContent)
                                        classes={
                                          Object {
                                            "root": "MuiCardContent-root",
                                          }
                                        }
                                      >
                                        <div
                                          className="MuiCardContent-root"
                                        >
                                          <WithStyles(ForwardRef(Typography))
                                            gutterBottom={true}
                                            variant="body1"
                                          >
                                            <ForwardRef(Typography)
                                              classes={
                                                Object {
                                                  "alignCenter": "MuiTypography-alignCenter",
                                                  "alignJustify": "MuiTypography-alignJustify",
                                                  "alignLeft": "MuiTypography-alignLeft",
                                                  "alignRight": "MuiTypography-alignRight",
                                                  "body1": "MuiTypography-body1",
                                                  "body2": "MuiTypography-body2",
                                                  "button": "MuiTypography-button",
                                                  "caption": "MuiTypography-caption",
                                                  "colorError": "MuiTypography-colorError",
                                                  "colorInherit": "MuiTypography-colorInherit",
                                                  "colorPrimary": "MuiTypography-colorPrimary",
                                                  "colorSecondary": "MuiTypography-colorSecondary",
                                                  "colorTextPrimary": "MuiTypography-colorTextPrimary",
                                                  "colorTextSecondary": "MuiTypography-colorTextSecondary",
                                                  "displayBlock": "MuiTypography-displayBlock",
                                                  "displayInline": "MuiTypography-displayInline",
                                                  "gutterBottom": "MuiTypography-gutterBottom",
                                                  "h1": "MuiTypography-h1",
                                                  "h2": "MuiTypography-h2",
                                                  "h3": "MuiTypography-h3",
                                                  "h4": "MuiTypography-h4",
                                                  "h5": "MuiTypography-h5",
                                                  "h6": "MuiTypography-h6",
                                                  "noWrap": "MuiTypography-noWrap",
                                                  "overline": "MuiTypography-overline",
                                                  "paragraph": "MuiTypography-paragraph",
                                                  "root": "MuiTypography-root",
                                                  "srOnly": "MuiTypography-srOnly",
                                                  "subtitle1": "MuiTypography-subtitle1",
                                                  "subtitle2": "MuiTypography-subtitle2",
                                                }
                                              }
                                              gutterBottom={true}
                                              variant="body1"
                                            >
                                              <p
                                                className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
                                              >
                                                Olion
                                              </p>
                                            </ForwardRef(Typography)>
                                          </WithStyles(ForwardRef(Typography))>
                                          <WithStyles(ForwardRef(Typography))
                                            color="textSecondary"
                                            variant="body2"
                                          >
                                            <ForwardRef(Typography)
                                              classes={
                                                Object {
                                                  "alignCenter": "MuiTypography-alignCenter",
                                                  "alignJustify": "MuiTypography-alignJustify",
                                                  "alignLeft": "MuiTypography-alignLeft",
                                                  "alignRight": "MuiTypography-alignRight",
                                                  "body1": "MuiTypography-body1",
                                                  "body2": "MuiTypography-body2",
                                                  "button": "MuiTypography-button",
                                                  "caption": "MuiTypography-caption",
                                                  "colorError": "MuiTypography-colorError",
                                                  "colorInherit": "MuiTypography-colorInherit",
                                                  "colorPrimary": "MuiTypography-colorPrimary",
                                                  "colorSecondary": "MuiTypography-colorSecondary",
                                                  "colorTextPrimary": "MuiTypography-colorTextPrimary",
                                                  "colorTextSecondary": "MuiTypography-colorTextSecondary",
                                                  "displayBlock": "MuiTypography-displayBlock",
                                                  "displayInline": "MuiTypography-displayInline",
                                                  "gutterBottom": "MuiTypography-gutterBottom",
                                                  "h1": "MuiTypography-h1",
                                                  "h2": "MuiTypography-h2",
                                                  "h3": "MuiTypography-h3",
                                                  "h4": "MuiTypography-h4",
                                                  "h5": "MuiTypography-h5",
                                                  "h6": "MuiTypography-h6",
                                                  "noWrap": "MuiTypography-noWrap",
                                                  "overline": "MuiTypography-overline",
                                                  "paragraph": "MuiTypography-paragraph",
                                                  "root": "MuiTypography-root",
                                                  "srOnly": "MuiTypography-srOnly",
                                                  "subtitle1": "MuiTypography-subtitle1",
                                                  "subtitle2": "MuiTypography-subtitle2",
                                                }
                                              }
                                              color="textSecondary"
                                              variant="body2"
                                            >
                                              <p
                                                className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
                                              >
                                                cat number: 
                                                787r
                                                 
                                                <br />
                                                 year: 
                                                1955
                                              </p>
                                            </ForwardRef(Typography)>
                                          </WithStyles(ForwardRef(Typography))>
                                        </div>
                                      </ForwardRef(CardContent)>
                                    </WithStyles(ForwardRef(CardContent))>
                                    <span
                                      className="MuiCardActionArea-focusHighlight"
                                    />
                                    <NoSsr>
                                      <WithStyles(undefined)
                                        center={false}
                                      >
                                        <ForwardRef(TouchRipple)
                                          center={false}
                                          classes={
                                            Object {
                                              "child": "MuiTouchRipple-child",
                                              "childLeaving": "MuiTouchRipple-childLeaving",
                                              "childPulsate": "MuiTouchRipple-childPulsate",
                                              "ripple": "MuiTouchRipple-ripple",
                                              "ripplePulsate": "MuiTouchRipple-ripplePulsate",
                                              "rippleVisible": "MuiTouchRipple-rippleVisible",
                                              "root": "MuiTouchRipple-root",
                                            }
                                          }
                                        >
                                          <span
                                            className="MuiTouchRipple-root"
                                          >
                                            <TransitionGroup
                                              childFactory={[Function]}
                                              component={null}
                                              exit={true}
                                            />
                                          </span>
                                        </ForwardRef(TouchRipple)>
                                      </WithStyles(undefined)>
                                    </NoSsr>
                                  </button>
                                </ForwardRef(ButtonBase)>
                              </WithStyles(ForwardRef(ButtonBase))>
                            </ForwardRef(CardActionArea)>
                          </WithStyles(ForwardRef(CardActionArea))>
                        </div>
                      </ForwardRef(Paper)>
                    </WithStyles(ForwardRef(Paper))>
                  </ForwardRef(Card)>
                </WithStyles(ForwardRef(Card))>
              </div>
            </li>
          </ForwardRef(GridListTile)>
        </WithStyles(ForwardRef(GridListTile))>
      </ul>
    </ForwardRef(GridList)>
  </WithStyles(ForwardRef(GridList))>
</Component>
`;
